cmd/hatchet-migrate/migrate/migrations/20250612173730_v1_0_25.sql

```diff
@@ -0,0 +1,9 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE INDEX v1_events_olap_scope_idx ON v1_events_olap (tenant_id, scope, seen_at DESC) WHERE scope IS NOT NULL;
```

Comment:
partial index to enable filtering by scope + seen at. one note here is we can't create this index concurrently unless we loop over all the partitions, but I figured this was probably fine